### ISSUE TITLE
Vehicle doors' physics can be broken

### ISSUE THREAD

**Issue by TacoDeBoss** [rwengine/openrw#70]
If slammed into a wall or other surface hard enough while still open, car door physics can be completely broken.
![screenshot_2016-05-23_22-12-57](https://cloud.githubusercontent.com/assets/2870898/15491198/c86665f2-2133-11e6-90f1-a132f3039a1d.png)

**Comment by parkerlreed** [comment]
How are you spawning vehicles? I enter "Test" and see a bunch of vehicle spawned commands in the terminal but I don't see a single vehicle.

**Comment by TacoDeBoss** [comment]
@parkerlreed They're all right there for me. Not sure how to help you with that problem.

**Comment by parkerlreed** [comment]
Yeah I immediately jumped off instead of turning around. All is good.

**Comment by TacoDeBoss** [comment]
It seems car doors actually detach and become invisible physics objects on the ground, but the door also stays attached to the car.
![screenshot_2016-05-23_22-34-05](https://cloud.githubusercontent.com/assets/2870898/15491497/8fcf585e-2136-11e6-82ef-90392da408d3.png)

**Comment by JayFoxRox** [comment]
(Removed offtopic discussion + a post by @TacoDeBoss about issue 72)

Is the original door issue still happening? I've never managed to reproduce it.

**Comment by dracc** [comment]
I've had this bug as late as yesterday, so I'd say yes, it's still there.
